pkg/utils/kubectl/kubectl.go

```diff
@@ -89,7 +89,7 @@ func CheckAllCommands(kubeconfigPath string, isContextSet bool, contextName stri
 			return errors.Wrapf(err, "parsing Kubernetes version string %q return by the EKS API server", version)
 		}
 		if version.Major == 1 && version.Minor < 10 {
-			return fmt.Errorf("Kubernetes version %s found, v1.10.0 or newer is expected with EKS %s", serverVersion, suggestion)
+			return fmt.Errorf("kubernetes version %s found, v1.10.0 or newer is expected with EKS %s", serverVersion, suggestion)
```

Comment:
just curious why this one is required, as the daily build seems passing for this file.

Reply:
I was using `staticcheck` directly, apparently `golang-ci` only uses a subset of its lint reports, I"ll remove these.